### ISSUE TITLE
Mesen-X on RetroArch

### ISSUE THREAD

**Issue by MineCreeper9898** [libretro/RetroArch#14383]
Mesen-X supports EPSM sound expansion

Is Mesen-X closed-source

Mesen-X is way different than Mesen

**Comment by hizzlekizzle** [comment]
No, it's not closed-source: https://github.com/NovaSquirrel/Mesen-X

However, it's not very different from regular Mesen, AFAICT, aside from the EPSG stuff and some GUI/translation commits.

**Comment by Ryunam** [comment]
I’ve been meaning to look into it for a while and I’m still planning to do that, however before attempting any work on Mesen-X I would like to finish the big core options upgrade that is in store for regular Mesen. 
Hopefully I can get that finished shortly.

**Comment by LibretroAdmin** [comment]
Just backport negativeexponents commits from his own Mesen libretro repo. For whatever reason he is not PRing them but it should probably contain most of these mapper additions. I did some of them but it's tedious and time consuming work since it's not completely in sync with our repo so it is still using vcxproj files, hence directly cherrypicking won't work and each git diff backport needs some additional touchups. 


If we add another core then we get the same criticisms we would get with all the snes cores, 'why so many? Which I Use? RA sucks blabla'. So we can truly never win here. Best to just keep building up our one Mesen core instead of splintering time and effort across several.

**Comment by Ryunam** [comment]
I am completely okay with either approach. Let me add however just two points, that I guess could be made in favor of having a separate core:

- Mesen-X seems to be, for all intents and purposes, both a continuation of the original and a separate entity of its own. Thus, keeping the two things separate in their libretro form would be entirely coherent relative to what happens already on the standalone front.
- We could in fact have Mesen-X as an upstream-friendly core, have it compiled directly from NovaSquirrel's repository with minimal edits on the libretro side, similarly to how bsnes-hd is built directly from Derkoun's repository. The regular Mesen core on the other hand could remain libretro-specific and retain all the work on customization and removal of unneeded components that was recently implemented.
